---
title: "Analysis Report 2: Gender differences in lung cancer"
author: "Emma Gibson"
date: "November 20, 2017"
output: github_document
bibliography: references.bib
csl: bioinformatics.csl
---

# Introduction

In today's society, cancer in general and lung cancer specifically is one of the most common causes of death. There have been numerous studies on the prevalance and risk factors of lung cancer, many of which agree that smoking presents a serious increase in risk for developing lung cancer. [@peto2000smoking] However, there have been conflicting results as to how biological sex affects risk of lung cancer. Some studies indicate that there there are no significant differences in the overall lung cancer rates between men and women, althought here might be differences in the rates of certain types of lung cancer. [@bain2004lung] However, other studies indicate that women are at a higher risk of developing all types of lung cancer. [@doi:10.1093/jnci/88.3-4.183] However, both of the cited studies agree with each other and other studies that adenocarcinoma rates are higher in females. [@devesa2005international] This type of cancer is characterized by regulation of several distinct genes, including those known to be associated with lung sulfricant proteins. [@taguchi2011lung]

# Methods

## Sample origin and sequencing

The original data that was analyzed in this report was gathered by Seo et al. in their paper studying the transcriptional landscape of lung cancer patients. [@seo2012transcriptional] Original tissue samples were taken surgically from cancer patients, and were extracted from both healthy and cancerous tissue. cDNA libraries were sequenced using an Illumina HiSeq machine. The goal of this initial study was to identify potential point mutations and alternative splicing in the genes of cancer patients.

After this initial study, Li et al. re-analyzed the sequence data looking for differences in gene expression between smoking and non-smoking cancer patients. [@li2015rna] Using various statistical analysis from R, such as Bionconductor and EdgeR, they compared overall gene transcription levels in smoking and non-smoking cancer patients. Their analysis excluded patients over age 75, and included additional data from 6 nonsmoker patients from another study.

## Computational

The data from the initial study was downloaded off of NCBI, then re-proscessed and re-mapped using several programs. First, trimmomatic was used to properly clean and proscess the sequences from the initial RNAseq dataset. [@doi:10.1093/bioinformatics/btu170] BioMart was used to download the annotated reference genomes that the RNA sequences would be mapped to. [@doi:10.1093/bioinformatics/bti525] This RNAseq information mapped to these reference genomes usingsailfish. [@patro_mount_kingsford_2014] Finally, the cleaned RNAseq data and its associated metadata was analyzed in R using extensions such as ggplott

# Results

```{r load-libraries, message = FALSE, echo = FALSE}
# Be sure to install these packages before running this script
# They can be installed either with the intall.packages() function
# or with the 'Packages' pane in RStudio

# load general-use packages
library("dplyr")
library("tidyr")
library("knitr")
library("ggplot2")
library("magrittr")

# this package allows for the easy inclusion of literature citations in our Rmd
# more info here: https://github.com/crsh/citr
# and here:
# http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
library("citr")
```

```{r load-data, message = FALSE, echo = FALSE}
# load the dataset from a compressed binary file
# it gets loaded as an object called "final_table"
# this has 3.4 million rows...so you will need to be thoughtful about
# how you analyze the data so that you don't overwhelm your laptop
load("output/final_compiled_counts/joined_count_data.RData")

# changes all columns names to lower case so Travis doesn't complain
names(final_table) %<>% tolower

# test that it loaded correctly before proceeding
stopifnot(exists("final_table"))
```

```{r make-summary-table, echo = FALSE}
# There are many many genes in this dataset, so we can
# subset it down to just a few here to look for interesting patterns
# in the most highly expressed
top_15 <- final_table %>%
  group_by(gender, genename) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count)) %>%
  head(n = 15)

# then we can use the `kable()` function to make a nicely formatted
# markdown table
top_15 %>%
  kable()
```

**Table 1.** The most highly expressed genes in both genders included *SFTPB* and *EEF1A1*.

```{r make-barplot-of-highly-expressed-genes, echo = FALSE}
# this code uses the same data as above, but use it to make a
# barplot - remember geom_col() is just like
# when you use geom_bar(stat = "identity")
top_15 %>%
  ggplot(aes(x = genename,
             y = mean_count,
             fill = gender)) +
  xlab("Gene name") +
  ylab("Mean read counts per gene") +
  theme_bw() +
  geom_col(position = "dodge")
```

**Figure 1.** Fifteen most common genes in male and female patients.

```{r make-boxplot-of-highly-expressed-genes, echo = FALSE}
# here we just want to pull out the unique gene names and turn
# them into a vector so we can use it below to make a boxplot
# we use the pull() funtion to get this as a vector, just like
# we did when making histograms several weeks ago
top_genes <- top_15 %>%
  ungroup() %>%
  select(genename) %>%
  unique() %>%
  pull()

# now we need to filter from the full data set again, because
# we don't just want summary data, we want all the data in
# order to make boxplots
final_table %>%
  filter(genename %in% top_genes) %>%
  ggplot(aes(x = genename,
             y = counts_lengthscaledtpm,
             fill = gender)) +
    geom_boxplot() +
    facet_wrap(~smoking_status) +
    xlab("Gene name") +
    ylab("Scaled read counts per gene") +
    ggtitle("Read counts per gene by gender and smoking status") +
    theme_bw() + # simplifies theme
    theme(axis.text.x = # rotates x axis labels vertically
            element_text(angle = 90,
                         hjust = 1))
```

**Figure 2.** Expression counts in top 15 genes, by gender and smoking status


```{r gender-and-smoking}
# first, this code finds various factors for each patient
gender_and_smoking_status <- final_table %>%
  group_by(smoking_status, cancer_stage, gender, sample_name_s) %>%
  count() %>%
  arrange(desc(n))

#next, this code interprets it as a dataframe and reads it into ggplot
with(gender_and_smoking_status, table(gender, smoking_status, cancer_stage)) %>%
  as.data.frame() %>% #to make it a ggplot-friendly dataframe
  ggplot(aes(x = smoking_status,
             y = Freq,
             fill = gender)) +
  facet_wrap(~cancer_stage) +
  geom_col(position = "dodge") +
  ggtitle("Smoking status of patients with each cancer stage") +
  xlab("Smoking status") +
  ylab("Number of patients") +
  theme_bw() +
  theme(axis.text.x =
            element_text(angle = 90,
                         hjust = 1))
```

**Figure 3.** Smoking statues of patients with each cancer stage by gender

```{r female-gene-table}
# this code finds the top 9 genes in female patients only
# I chose 9 instead of 15 because the original top 15 had
# 9 genes because some were common in both genders
top_9_f <- final_table %>%
  filter(gender == "female") %>%
  group_by(genename) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm)) %>%
  arrange(desc(mean_count)) %>%
  head(n = 9)

#next, this line puts it into a nice markdown table
kable(top_9_f)
```

**Table 2.** Top 9 genes in female patients

```{r female-gene-graph}
# this puts the top 9 female genes into a facet-friendly format
top_genes_f <- top_9_f %>%
  ungroup() %>%
  select(genename) %>%
  unique() %>%
  pull()

# this code makes a figure similar to Figure 2, but with top female genes
# it doesn't plot top_genes_f because I wanted to include male patients
final_table %>%
  filter(genename %in% top_genes_f) %>%
  ggplot(aes(x = genename,
             y = counts_lengthscaledtpm,
             fill = gender)) +
    geom_boxplot() +
    facet_wrap(~smoking_status) +
    xlab("Gene name") +
    ylab("Scaled read counts per gene") +
    ggtitle("Read counts per gene of genes common to female smokers") +
  theme_bw() +
  theme(axis.text.x =
            element_text(angle = 90,
                         hjust = 1))
```

**Figure 4.** Expression counts in top 9 most common female genes, by gender and smoking status

```{r smoking-in-female-genes}
# this gets the appropriate metadata to make a figure faceted by smoking status
top_f_in_all_status <- final_table %>%
  filter(genename %in% top_genes_f) %>%
  group_by(gender, genename, smoking_status) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm))

top_f_in_all_status %>%
  ggplot(aes(x = genename,
             y = mean_count,
             fill = gender)) +
  facet_wrap(~smoking_status) +
  geom_col(position = "dodge") +
  ggtitle("A. Expression of most common genes for female
          patients in all patients by smoking status") +
  xlab("Gene name") +
  ylab("Mean read counts per gene") +
  theme_bw() +
  theme(axis.text.x =
            element_text(angle = 90,
                         hjust = 1))

# this gets the appropriate metadata to make a figure faceted by cancer stage
top_f_in_all_stage <- final_table %>%
  filter(genename %in% top_genes_f) %>%
  group_by(gender, genename, cancer_stage) %>%
  summarize(mean_count = mean(counts_lengthscaledtpm))

top_f_in_all_stage %>%
  ggplot(aes(x = genename,
             y = mean_count,
             fill = gender)) +
  facet_wrap(~cancer_stage) +
  geom_col(position = "dodge") +
  ggtitle("B. Expression of most common genes for female
          patients in all patients by cancer stage") +
  xlab("Gene name") +
  ylab("Mean read counts per gene") +
  theme_bw() +
  theme(axis.text.x =
            element_text(angle = 90,
                         hjust = 1))
```

**Figure 5.** Abundance of top 9 female genes by gender, faceted by A. smoking status and B. cancer stage

# Discussion

Initially, I noticed that gene expression was relatively even in both genders for the most common genes, witht he exception of EEF1A1 and SFTB (Figure 1). After breaking further breaking down this data by smoking status, I found a similar correlation between gene expression between the genders, but I noticed that there seemed to be less data on female smokers (Figure 2). When I looked into this further, I found that most of the female patients had never smoked and were in early stages of cancer. (Figure 3) Given this information, I decided to investigate whether the most common genes in female patients might be different than the most common genes in all patients, and found that they were (Table 2). Most of the genes tht were most commom in female patients were the same as the most common in general patients, and they showed similar expression patterns as well (Figure 4) Lastly, I decided to examine the expression of these genes compared to smoking status and cancer stage, and found that the genes that were more common in females were also most common in nonsmokers with early cancer stages, which makes sense given the other analyses (Figure 5).

Of the few genes that are consistently more common in female patients than males are genes like SFTBP, which seems to be especially common in cancer stage 1B (Figure 5B). Although it is quite common in both genders at this stage, SFTBP is the most common gene in females at this stage, but not in males. Although this could be due to the gene simply being expressed in early stages of cancer, the difference could potentially hint that a different type of cancer common to females at this stage. Studies have shown that the SFTBP gene is associated with adenocarcinoma, the type of lung cancer that is especially common in females. [@taguchi2011lung] Given that Li et al. showed that there are differences in gene expression among lung cancer patients who do and do not smoke, it is possible that adenocarcinomaadenocarcinoma might be more common in people who do not smoke, rather than females. [@li2015rna] However, this gene is still more common in females who have never smoked than males who have never smoked, indicating that SFTBP's abundance in female patients is more likely due to gender than smoking status.

# Sources Cited
